frontend/src/components/bucket/BucketList.vue

```diff
@@ -86,8 +86,17 @@ onMounted(async () => {
           {{ bucketConfigTitle }}
         </h3>
 
+        <Message severity="info">
+          Need a bucket? Or help with buckets? Contact <a
+            href="https://apps.nrs.gov.bc.ca/int/jira/servicedesk/customer/portal/1/create/701"
+            target="_blank"
+          >NRIDS Optimization</a> (Natural Resource ministries) or your ministry's service desk.
```

Comment:
The rhetorical questions here are a bit jarring imo. It may be better to write this as an assertion
```
Please contact NRIDS Optimization (Natural Resource ministries) or your ministry's service desk if you need help with buckets.
```

Reply:
I agree, this works.